src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -159,24 +198,26 @@ def set_attrs(self, path, size, isolate_namespace, pool, uid, gid):
                 raise VolumeException(-e.args[0], e.args[1])
 
         # set uid/gid
+        uid = attrs.get("uid")
         if uid is None:
             uid = self.group.uid
         else:
             try:
-                uid = int(uid)
```

Comment:
This  has introduced the regression,
https://pulpito.ceph.com/pdonnell-2020-08-27_01:15:36-fs-wip-pdonnell-testing-20200826.234047-nautilus-distro-basic-smithi/5379256/